int32 human_id
float64 C0
float64 C1
float64 C2